_test/check-configlet-fmt.sh

```diff
@@ -9,6 +9,25 @@ configlet="${repo}/bin/configlet"
 c="${repo}/config.json"
 m="${repo}/config/maintainers.json"
 
+# Check if config.json or maintainers.json were modified
+current_branch_name="$(git rev-parse --abbrev-ref HEAD)"
+
+check_pattern="config.json\|config/maintainers.json"
```

Comment:
something I don't care to have fixed: the . are going to match any character, which means this will fire if we change a file named configxjson

I don't want to sacrifice clarity to have this fixed because I find it unlikely we will ever have a file with that name.